### DISCUSSION TITLE
Get notified when a repository is archived

### DISCUSSION THREAD

**Discussion by brandones** [community/community#21576] [New to GitHub]
<p>This is a feature request. I’d like a mode of “watching” a repository where I only get notified if the repository if archived. This would be useful for projects that I depend on just enough that I would like to know if they are no longer going to be maintained.</p>

**Comment by tuves** [comment, marked as answer]
<p>Hello <a class="mention" href="/brandones">@brandones</a> and welcome to the GitHub Community! :smiley: Please drop all feedback ideas and feedback <a href="https://support.github.com/contact/feedback">here</a> so that they may be tracked and routed to the appropriate teams.</p>
